content/docs/command-reference/remote/modify.md

```diff
@@ -108,6 +108,18 @@ these settings, you could use the following options:
   $ dvc remote modify myremote endpointurl https://myendpoint.com
   ```
 
+- `access_key_id` - AWS access key id to use to access S3:
```

Comment:
```suggestion
- `access_key_id` - AWS Access Key ID:
```

Reply:
But does this need an explanation? When is an access key ID (and secret) needed?